.github/workflows/gradle.yml

```diff
@@ -0,0 +1,35 @@
+# This workflow will build a Java project with Gradle
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: Java CI with Gradle
+
+on:
+  push:
+    branches: '**'
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
```

Comment:
Will it work with higher versions of java? like java 11 or 15?

Reply:
If we avoid using specific feature from java 11 or 15, we should be ok. However, we can consider to upgrade the version we are using in the future